plugins/polio/js/src/forms/DestructionsForm.tsx

```diff
@@ -0,0 +1,108 @@
+import { useFormikContext } from 'formik';
+import React, {
+    FunctionComponent,
+    useCallback,
+    useEffect,
+    useState,
+} from 'react';
+// @ts-ignore
+import { useSafeIntl } from 'bluesquare-components';
+import { Box, Button, Grid } from '@material-ui/core';
+import MESSAGES from '../constants/messages';
+import { DestructionForm } from './DestructionForm';
+
+type Props = {
+    round: any;
+    accessor: string;
+};
+
+export const DestructionsForm: FunctionComponent<Props> = ({
+    round,
+    accessor,
+}) => {
+    const { formatMessage } = useSafeIntl();
+    const { setFieldValue } = useFormikContext();
+    const { destructions = [] } = round ?? {};
+    const [enableRemoveButton, setEnableRemoveButton] =
+        useState<boolean>(false);
+
+    const lastIndex = destructions.length >= 1 && destructions.length - 1;
+
+    const handleAddShipment = () => {
```

Comment:
This method should be renamed to "handleAddDestruction"

Reply:
Me copy/pasting my own code?😅